### PR TITLE
Test/form

### PR DESCRIPTION
**Pull request by lilianaweimer**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- Didn't have any tests before

## What is the new behavior?

- 2 passing unit tests for Form, one not-quite-passing test for Form

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

We are unsure of how to test changes in a select/dropdown menu, so we will get that test working later. (After we ask for help.)

### CODE REVIEW

File: src/Form/Form.test.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import '@testing-library/jest-dom/extend-expect';
+import Form from './Form';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux'
+
+const store = createStore(() => ({playerName: 'test', questions: 'test'}))
+
+
+describe('Form', () => {
```
Comment (Billwilke42) [inline src/Form/Form.test.js:14]: Nice job getting the first tests going!

File: src/Form/Form.test.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                <Provider store={store}>
+                    <Form />
+                </Provider>
+            </BrowserRouter>);
+        
+
+      const header = getByText('Welcome to BlahBlah');
+
+      expect(header).toBeInTheDocument();
+    });
```
Comment (KyleWong2510) [inline src/Form/Form.test.js:28]: Good to know that we need to wrap in the Provider

File: src/Form/Form.test.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    it('should be able to fill the form out', () => {
+        const { getByTestId, getByPlaceholderText, getByDisplayValue, debug } = render(
+            <BrowserRouter>
+                <Provider store={store}>
+                    <Form />
+                </Provider>
+            </BrowserRouter>);
+
+        const nameInput = getByPlaceholderText('Name');
+        const roundOne = getByTestId('round1');
```
Comment (Billwilke42) [inline src/Form/Form.test.js:67]: good thinking adding a test id

File: src/Form/Form.test.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+        expect(nameInput).toBeInTheDocument();
+        expect(roundOne).toBeInTheDocument();
+        expect(roundTwo).toBeInTheDocument();
+        expect(roundThree).toBeInTheDocument();
+        expect(roundFour).toBeInTheDocument();
+        expect(roundFive).toBeInTheDocument();
+        expect(roundSix).toBeInTheDocument();
+        expect(playButton).toBeInTheDocument();
+
```
Comment (KyleWong2510) [inline src/Form/Form.test.js:55]: Nice job passing initial render!

File: src/Form/Form.test.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+        });
+        // debug()
+
+        const changedName = getByDisplayValue('somebody');
+        const changedRoundOne = getByDisplayValue('History');
+
+        expect(changedName).toBeInTheDocument();
+        expect(changedRoundOne).toBeInTheDocument();
+    });
+
```
Comment (KyleWong2510) [inline src/Form/Form.test.js:103]: Need to figure out how to fireEvents on the right inputs to know which one is selected.

**Pull request merged**